migrations/819-deleted-addons-nullify-guid.py

```diff
@@ -1,23 +1,4 @@
 """Deleting add-ons is in fact soft deleting them, and emptying some of its
 fields with a unique constraint (like the slug).
-We now also empty the GUID field, to allow re-submitting those add-ons (once
-the GUID has been removed from the BlacklistedGuid through the admin.
 
-This migration empties all the previously soft deleted add-ons."""
-
-from django.conf import settings
-
-import amo
-from addons.models import Addon
-from users.models import UserProfile
-
-
-addons = Addon.unfiltered.no_cache().filter(status=amo.STATUS_DELETED,
-                                            guid__isnull=False)
-user = UserProfile.objects.get(pk=settings.TASK_USER_ID)
-amo.set_user(user)
-for addon in addons:
-    amo.log(amo.LOG.DELETE_ADDON, addon.pk, addon.guid, addon,
-            created=addon.modified)
-    addon.guid = None
-    addon.save()
+Now we don't clear the GUID so this migration is unneeded."""
```

Comment:
I don't think we update previous migrations. I'd rather leave this one untouched.